tensorflow/go/operation_test.go

```diff
@@ -222,6 +222,10 @@ func TestOperationConsumers(t *testing.T) {
 			t.Fatalf("%d. Got op name %q, wanted %q", i, got, want)
 		}
 	}
+
+	if b.Consumers() != nil {
```

Comment:
Do we want to distinguish between `len(b.Consumers()) == 0` and `nil`? I don't think so, so let's just make this:

```go
if len(b.Consumers()) == 0 {
  t.Fatalf(...)
}
```